simplq/src/styles/homePage.module.scss

```diff
@@ -112,15 +112,33 @@
 }
 
 .extra-info {
-  @include background-color();
-  @include text-color();
-  @include center-vertically();
-  @include center-horizontally();
-  div {
-    display: flex;
-    align-items: center;
-    p {
-      min-width: 10rem; // So that the text alings along a vertical line, as the entre component in vertically centered
-    }
+  background-color: $primary-color-dark;
+  padding: 3rem;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  text-align: center;
+  align-items: flex-start;
+
+  .card {
+    padding: 0.5rem;
+    max-height: 16rem;
+    max-width: 16rem;
+  }
+
+  img {
+    width: 20%;
+    height: 20%;
+  }
+
+  .brief-description {
+    font-weight: bold;
+    color: white;
```

Comment:
I think we should do a `text-transform: capitalize` here. It's not in design, but looks better.